services.py

```diff
@@ -21,7 +21,14 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
 services = {
   # service: (should_log, frequency, qlog decimation (optional))
   # note: the "EncodeIdx" packets will still be in the log
-  "sensorEvents": (True, 100., 100),
+  "sensorEventsDEPRECATED": (True, 100., 100),
+  "gyroscope": (True, 104., 100),
+  "gyroscope2": (True, 100., 100),
+  "accelerometer": (True, 104., 100),
```

Comment:
update the qlog decimation to be 104 as well

Reply:
updated